llvm/include/llvm/Transforms/Utils/Cloning.h

```diff
@@ -207,6 +207,15 @@ void CloneAndPruneFunctionInto(Function *NewFunc, const Function *OldFunc,
                                const char *NameSuffix = "",
                                ClonedCodeInfo *CodeInfo = nullptr);
 
+/// Process function's subprogram attachment to collect relevant debug
+/// information in DIFinder.
```

Comment:
Could you elaborate on "relevant debug info"? Reading this comment, I can't quite tell what the function is doing